src/ImageRunModal.jsx

```diff
@@ -893,7 +893,7 @@ export class ImageRunModal extends React.Component {
                               <Popover aria-label={_("Restart policy help")}
                                 enableFlip
                                 bodyContent={this.props.userLingeringEnabled ? _("Restart policy to follow when containers exit. Using linger for auto-starting containers may not work in some circumstances, such as when ecryptfs, systemd-homed, NFS, or 2FA are used on a user account.") : _("Restart policy to follow when containers exit.")}>
-                                  <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                  <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1322-20230703-044046-64a79ae1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)